Jenkinsfile

```diff
@@ -198,7 +198,6 @@ pipeline {
             trainer.precision=16 \
             trainer.amp_level=O1 \
             trainer.gpus=[0] \
-            +trainer.fast_dev_run=true \
```

Comment:
why is this gone now?

Reply:
because squad cannot be evaluated for steps, but only on the entire evaluation dataset.